test/test_runner.c

```diff
@@ -361,6 +363,9 @@ void CB2_TestRunner(void)
         MgbaExit_(gTestRunnerState.exitCode);
         break;
     }
+
+    if (gMain.callback2 == CB2_TestRunner)
+        goto top;
```

Comment:
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/2904965/b1c210d3-f24f-483c-b6a0-0de9de60887a)

Reply:
Haha! I was discussing this with jiang, and I think they found that simply doing `gMain.callback2();` at the end didn't work. I can't remember if we tried `if (gMain.callback2 == CB2_TestRunner) CB2_TestRunner();`.

You'd think that either of those should work, provided that GCC does tail-call elimination (if it doesn't, then it'll blow the stack).